packages/patternfly-4/react-catalog-view-extension/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@patternfly/react-catalog-view-extension",
```

Comment:
do we need to include the word `extension`? I think being in its own package is enough to get the point across.

Reply:
Yes - that is the pattern we need to follow for any extension package so we can clearly understand the intent behind the package.